src/utils/network_calculations/ybus_calculations.jl

```diff
@@ -117,11 +117,10 @@ function _buildybus(branches, nodes, fixed_admittances)
         if get_name(b) == "init"
             throw(DataFormatError("The data in Branch is invalid"))
         end
-
-        _ybus!(ybus, b, num_bus)
+        get_available(b) && _ybus!(ybus, b, num_bus)
```

Comment:
In the existing code `ix` is unused and can be deleted. Also, (just a comment) the exception for `init` seems non-ideal since it is tied to the name and not data values.